src/policy/fees.h

```diff
@@ -241,34 +243,32 @@ class CBlockPolicyEstimator
     };
 
     // map of txids to information about that transaction
-    std::map<uint256, TxStatsInfo> mapMemPoolTxs;
+    std::map<uint256, TxStatsInfo> mapMemPoolTxs GUARDED_BY(m_cs_fee_estimator);
 
     /** Classes to track historical data on transaction confirmations */
     std::unique_ptr<TxConfirmStats> feeStats;
     std::unique_ptr<TxConfirmStats> shortStats;
     std::unique_ptr<TxConfirmStats> longStats;
```

Comment:
I'm curious - did you try marking these `PT_GUARDED_BY`? Seems it would be appropriate, based on the lock in `estimateRawFee`.